src/scss/components/list/_index.scss

```diff
@@ -1,34 +1,36 @@
-@import (reference) 'components/grid/index';
+@import "components/grid/index";
 
 #List {
-  .reset() {
+  @mixin reset() {
```

Comment:
FYI: This syntax doesn't work in SCSS. Mixins have to be defined at the top-most level (don't nest mixins).